build/commands/lib/config.js

```diff
@@ -523,7 +523,7 @@ Config.prototype.buildArgs = function () {
     args.android_channel = this.channel
     if (!this.isReleaseBuild()) {
```

Comment:
This change will only reflect in non release builds

btw why is this needed and what might be impacted by changing this?

Reply:
change package name for all builds